src/code42cli/cmds/devices.py

```diff
@@ -103,17 +106,28 @@ def _change_device_name(sdk, guid, name):
 
 @devices.command()
 @device_guid_argument
-@format_option
+@click.option(
+    "-f",
+    "--format",
+    type=click.Choice(
+        [OutputFormat.TABLE, OutputFormat.JSON, OutputFormat.RAW], case_sensitive=False
```

Comment:
CSV format doesn't make sense for `devices show`, so needed to tweak choices available.

Reply:
Typically, for show commands, we were just using Table

Reply:
Ah, true. That simplifies things.